src/formElementPlugins/textInfo/src/ApplicationView.tsx

```diff
@@ -2,13 +2,53 @@ import React from 'react'
 import { Message } from 'semantic-ui-react'
 import { ApplicationViewProps } from '../../types'
 
+interface TextInfoProps {
+  parameters: any
+  Markdown: any
+}
+
+// Extract main content to another Component so it can be shared with SummaryView (and TemplateView)
+export const TextInfoElement: React.FC<TextInfoProps> = (props) => {
+  const { parameters, Markdown } = props
+  const { title, text, style } = parameters
+  if (['basic', 'info', 'warning', 'success', 'positive', 'negative', 'error'].includes(style)) {
```

Comment:
Maybe nice to define a type for this:
`type StyleType = 'basic' | 'info' | 'warning' | 'success' | 'positive' | 'negative' | 'error'`

And then change to 

```suggestion
  const { title, text } = parameters
  const style: StyleType = parameters.style
  if (style) {
```